src/django/pfb_analysis/models.py

```diff
@@ -43,8 +43,11 @@ def __repr__(self):
 
     def save(self, *args, **kwargs):
         """ Override to do validation checks before saving, which disallows blank state_abbrev """
+        if not self.label:
+            self.label = slugify(self.name)
```

Comment:
Borrowed behavior from the existing `Organization` model, which has reversed `name` and `label` text/slug types. These should be consistent; looks like `Organization` is the one that's backwards.